components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -246,6 +246,19 @@ sealed class ContentAction : BrowserAction() {
      * Removes the [SearchRequest] of the [ContentState] with the given [sessionId].
      */
     data class ConsumeSearchRequestAction(val sessionId: String) : ContentAction()
+
+    /**
+     * Updates the [fullScreenEnabled] with the given [sessionId].
+     */
+    data class FullScreenChangedAction(val sessionId: String, val fullScreenEnabled: Boolean) : ContentAction()
+
+    /**
+     * Updates the [layoutInDisplayCutoutMode] with the given [sessionId].
+     *
+     * layoutInDisplayCutoutMode value is defined in
```

Comment:
nit: `@param` so that the KDocs render correctly. :)

Reply:
Agreed.  Thanks,